.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+
+Please include a summary of the changes. Also, please include relevant motivation and context. Remember to label your pull request properly, of which the type of change should be included; **bug-fix**, **new-feature**, **enhancement**, **docs** or **breaking-change**.
+
+# Result
+
+If you've made visual changes, please include **before** and **after** images, preferably with a description.
```

Comment:
Can we add a quick line about not including real user information in the images? I know it's quite obvious for most Webkommers now, but I remember doing so when I was new and nobody had told me not to... It wouldn't hurt anybody, but could help someone with a friendly reminder:)